data/human/news.txt

```diff
@@ -2190,3 +2190,342 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+
+news "High Tension Situation"
+	location
+		attributes "ccor"
+	name
+		word
+			"Standoff"
+	message
+		word
+			"On the corner "
+			"By a landing pad "
+			"At the entrance to the spaceport market, "
+		word
+			"some CCOR guerrillas are "
+			"there's a group of people in CCOR colors "
+		word
+			"distributing pamphlets"
+			"handing out literature"
+			"delivering aid"
+			"rallying against the local authorities"
+			"supporting local protests"
+		word
+			". "
+		word
+			"When "
+		word
+			"the planetary authorities "
+			"a gang of local pirates "
+		word
+			"accost "
+			"confront "
+		word
+			"them and demand that they "
+		word
+			"disperse"
+			"make themselves scarce"
+			"abandon their objective"
+			"shut up and buzz off"
+		word
+			", "
+		word
+			"the guerrillas "
+			"the group "
+		word
+			"bluster at them "
+			"saber-rattle "
+		word
+			"in response"
+		word
+			", but they end up dispersing."
+			", and the two groups back down."
+
+phrase "Pronoun"
+	word
+		"his"
+		"her"
+		"their"
+
+phrase "Careless Ends Well"
+	word
+		"turns around to reenter the ship"
+	word
+		" as it's struck by a wave, nudging it close to the edge of the platform. The captain then moves the ship to a lowerable pad."
+		", moving it to a lowerable pad before it could be caught by a wave."
+
+phrase "Careless Ends Poorly"
+	word
+		"ignores air traffic control and heads to the bar. The ship"
+		"argues with the controller about their landing. The ship"
+		"turns around to reenter the ship. However, it"
+	word
+		" is "
+	word
+		"struck "
+		"caught "
+	word
+		"by a wave and "
+	word
+		"swept into the sea."
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet "Jakobsen"
+	name
+		word
+			"Recklessness"
+	message
+		word
+			"A Sparrow"
+			"An Enforcer"
+			"A Shuttle"
+			"A Heavy Shuttle"
+			"A Star Barge"
+			"A Berserker"
+			"A Fury"
+			"A Hawk"
+			"A Wasp"
+			"A Scrapper"
+		word
+			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} ship. The captain "
```

Comment:
This is so we don't get repetitions of the word 'ship' in case `"turns around to reenter the ship. However, it"` is the next line.
```suggestion
			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} vessel, but the captain "
```